src/nvim/getchar.c

```diff
@@ -1465,9 +1465,10 @@ int vgetc(void)
        * converted character.
        * Note: This will loop until enough bytes are received!
        */
-      if (has_mbyte && (n = MB_BYTE2LEN_CHECK(c)) > 1) {
+      buf[0] = (char_u)c;
+      i = 1;
```

Comment:
this is an odd place to put this, because it's done again on line 1472. Perhaps this should go in a `else {}` block?